package.json

```diff
@@ -59,5 +63,8 @@
     "ts-node": "10.9.1",
     "typechain": "8.1.0",
     "typescript": "4.8.4"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.8.0"
```

Comment:
all dependencies should be devDependencies for this project

Reply:
done.